Source/WebInspectorUI/UserInterface/Views/HeapSnapshotDataGridTree.js

```diff
@@ -254,7 +254,7 @@ WI.HeapSnapshotInstancesDataGridTree = class HeapSnapshotInstancesDataGridTree e
     }
 };
 
-WI.HeapSnapshotObjectGraphDataGridTree = class HeapSnapshotInstancesDataGridTree extends WI.HeapSnapshotDataGridTree
```

Comment:
What??

Reply:
this was a drive-by as i was having a hard time using "Jump to Symbol" with this because it was named the same as `WI.HeapSnapshotInstancesDataGridTree`